test/testvm.py

```diff
@@ -714,6 +709,12 @@ def _start_qemu(self, maintain=False, tty=False, monitor=None):
         drive_to_start = self._image_root
         if os.path.exists(self._image_image):
             drive_to_start = self._image_image
+        elif os.path.exists(self._image_original):
+            subprocess.check_call([ "qemu-img", "create", "-q",
+                                    "-f", "qcow2",
+                                    "-o", "backing_file=%s" % self._image_original,
```

Comment:
> Otherwise stuff like this will break.

Does it? There is no shell parsing involved here. `qemu-img` will see the string "backing_file=I have a space" somewhere in argv and will hopefully do the right thing.